index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE>
+<html>
+
+<head>
+    <title>Code Your Future | CYF</title>
+    <link rel="stylesheet" href="styles/style.css">
+</head>
+
+<body>
+    <!-- Site Header-->
+    <div class="site-header">
+        <h1 class="website-title">CYF</h1>
+        <!-- Navigate bar-->
+        <div class="navbar">
+            <button id="Menu 1" type="button" class="btn btn-primary">Menu 1</button>
```

Comment:
Be careful with spaces in your IDs and classnames - the browser will treat them as two separate ones (a) `Menu` and (b) `1`